ocis/pkg/runtime/service/service.go

```diff
@@ -41,9 +41,9 @@ import (
 	search "github.com/owncloud/ocis/extensions/search/pkg/command"
 	settings "github.com/owncloud/ocis/extensions/settings/pkg/command"
 	sharing "github.com/owncloud/ocis/extensions/sharing/pkg/command"
-	storagemetadata "github.com/owncloud/ocis/extensions/storage-metadata/pkg/command"
 	storagepublic "github.com/owncloud/ocis/extensions/storage-publiclink/pkg/command"
 	storageshares "github.com/owncloud/ocis/extensions/storage-shares/pkg/command"
+	storageSystem "github.com/owncloud/ocis/extensions/storage-system/pkg/command"
```

Comment:
why `storageSystem` ? Other imports don't use camel case

Reply:
I think @wkloucek started introducing that in the previous config env PR. Since it is internal we can change it after beta.